### DISCUSSION TITLE
tracy-session is not marked with secure flag

### DISCUSSION THREAD

**Discussion by vavra** [nette/tracy#571] [General]
Why there is no secure flag in response header

	Set-Cookie: tracy-session=49ac228f0d; expires=Thu, 05 Sep 2024 18:13:57 GMT; Max-Age=31557600; path=/; HttpOnly
	
for tracy-session?

bin/vendor/tracy/tracy/src/Tracy/Session/FileSession.php
line #54
setcookie($this->cookieName, $id, time() + self::CookieLifetime, '/', '', false, true);

If there was the penultimate parametr true or computed from $_SERVER, it could be.
The Burp tool is complaining about it.

"Issue detail
The following cookie was issued by the application and does not have the secure flag set:
    tracy-session
 "

**Comment by dg** [comment]
Send PR please
